app/views/content/life-as-a-teacher/teaching-as-a-career/making-a-career-change-to-teaching/_article.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <p>I spent 27 years working in finance in London. Having raised my own family and enjoyed a successful career in the city, I wanted to pursue one of my long-term ambitions, and chose to make the switch to teaching.</p>
+  </section>
+  <section class="col col-720">
+    <%= render Content::QuoteComponent.new(
+    text: "I’d been working in the same role for many years and I was no longer feeling challenged. So I had a decision to make: stay in my finance role, or give myself a new break.",
+    large: true
+  ) %>
+  </section>
+  <section class="col col-720 col-space-l-top">
+    <p>I’d gained a lot of experience of working with people who have different skill sets and approaches, and it really helped me when faced with a classroom of students all of whom have different approaches, ideas and attitudes.</p>
+    <p>Having worked with adults all my life, I was used to dealing with people who are prepared to sit and listen to you. Now that I’m working with children, I really have to earn their respect. It’s extremely rewarding to have a student come out of a lesson and thank you, or receive an email from a parent telling you how much impact you’ve had on their child’s learning.</p>
+    <p>Of course, there are highs and lows in teaching, but the highs are immeasurable. You might have a child who starts the lesson not particularly focused because they’re struggling with the subject. As you guide them and encourage them, their understanding increases and all of a sudden they’ve got their hand up to answer a question. That light-bulb moment really gives you a buzz.</p>
```

Comment:
```suggestion
    <p>Of course, there are highs and lows in teaching, but the highs are immeasurable. You might have a child who starts the lesson not particularly focused because they’re struggling with the subject. As you guide them and encourage them, their understanding increases and all of a sudden they’ve got their hand up to answer a question. That lightbulb moment really gives you a buzz.</p>
```